lib/sequences/05_openid_connect_sequence.rb

```diff
@@ -0,0 +1,111 @@
+class OpenIDConnectSequence < SequenceBase
```

Comment:
Add sequence self-test.  We need to have all these done this week, so might as well get that in here now.